types/metro-resolver/errors.d.ts

```diff
@@ -0,0 +1,37 @@
+import { FileAndDirCandidates, FileCandidates } from './types';
+
+export class FailedToResolvePathError extends Error {
```

Comment:
@afoxman, did we have a policy of mirroring the folder structure of Metro? Should these be in separate files?

Reply:
ping @afoxman

Reply:
Hi @bolasblack, thanks for updating these types! And yes, let's try to keep mirroring the folder structure for now. I see pros and cons to this, but it's been helpful so far, and hasn't been a huge burden.

Reply:
Sorry for the late reply, I will update the folder structure

Reply:
@afoxman @tido64 Hi guys, I have finished the structure updating, please take a look again